### DISCUSSION TITLE
1.8.1

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#973] [Обновления]
- Добавлена поддержка Loom (#946, #34)

  ^ субтиры от сайта не работают до следующего обновления

- Добавлена поддержка домена `madmaxworld.tv` для BannedVideo
- Убрана отправка уведомления о переводе, если перевод уже существует ([ee3a66d0b6ef3c6af7b61136dae0c65df0d74337#commitcomment-150456758](https://github.com/ilyhalight/voice-over-translation/commit/ee3a66d0b6ef3c6af7b61136dae0c65df0d74337#commitcomment-150456758))
- Исправлена работа с некоторыми плейлистами VK Video (если ссылка содержала слитно написанный видео ID) (#971)

## What's Changed
* Update 1.8.1 by @ilyhalight in https://github.com/ilyhalight/voice-over-translation/pull/972

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.8.0...1.8.1

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.8.1'>1.8.1</a>.</em>

**Comment by retmo** [comment]
перестал работать ,пишет длинное видео ,хотя оно 4 мин .

**Reply by ilyhalight** [reply]
дайте ссылку на видео

**Reply by retmo** [reply]
Если подробнее. Я смотрю на яндекс диске по ссылке. Ссылку не могу дать ,приватное видео ).Я ради эксперимента, откатился на 1.7.2 и все работает как нужно.)

**Reply by retmo** [reply]
и я пробовал разные видео, везде одинаково)

**Reply by ilyhalight** [reply]
> Я ради эксперимента, откатился на 1.7.2 и все работает как нужно

но ведь с 1.7.2 ничего связанного с яндекс диском толком не менялось в расширение...

В дев ветке исправил

**Reply by V3mp812** [reply]
почему то на видео по ссылке с ютуба начал писать доступ к видео ограничен

**Reply by ilyhalight** [reply]
> почему то на видео по ссылке с ютуба начал писать доступ к видео ограничен

значит 18+ на видео, либо еще что-то что мешает яндексу перевести
